### ISSUE TITLE
remove dependence on apivore

### ISSUE THREAD

**Issue by erluti** [department-of-veterans-affairs/va.gov-team#7572]
## Problem Statement
How do we remove our dependency on apivore?  It is no longer maintained and doesn't support Rails 6.

## Objectives
* Use only gems that support upgrading our system
---
## Tasks
- [ ] Review/Maintain test coverage provided from using apivore
## Acceptance Criteria
- [ ] no longer dependent on apivore

**Comment by erluti** [comment]
Apivore is only used by `spec/request/swagger_spec.rb`.  This spec makes requests to endpoints and compares responses to the swagger documentation.  We can remove apivore with no issues if there's a way to set up an equivalent test.

**Comment by erluti** [comment]
I looked into using the `committee` gem because it has an `assert_schema_conform` test helper.  It was failing to load the driver from our specs because of how we use `#ref` in the parameters.
WIP branch: https://github.com/department-of-veterans-affairs/vets-api/tree/be-tools/pb/use-committee-to-test-api-specs

Issue I created related to the driver failing to load our schema https://github.com/interagent/committee/issues/259 for the committee gem.
